server/src/defaultConfig.json

```diff
@@ -7,5 +7,13 @@
   "colorPalette": "#0078d4",
   "waitingTitle": "Thank you for choosing Lamna Healthcare",
   "waitingSubtitle": "Your clinician is joining the meeting",
-  "logoUrl": ""
+  "logoUrl": "",
+  "postCall": {
+    "survey": {
+      "type": "",
+      "options": {
+        "surveyUrl": ""
```

Comment:
Setting `surveyUrl` here seems odd. This options object with surveyUrl implements a specific interface (`MSFormsSurveyOptions` and `ThirdPartySurveyOptions`), and yet we are not setting the type as MS Forms or Third Party. Is it more appropriate to set `options` as null or undefined? Or set the entire `postCall` as null or undefined?